cmd/pbstatuser/main.go

```diff
@@ -41,7 +42,20 @@ type SourceInfo struct {
 	Identity identity.XRHID
 }
 
+const (
+	metricNameSourceAvailabilityCheckRequestTotal = "source_availability_check_request_total"
+)
+
 var (
+	totalAvailabilityCheckReqs = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name:        metricNameSourceAvailabilityCheckRequestTotal,
+			Help:        "availability Check requests count partitioned by type (aws/gcp/azure), source status and error",
+			ConstLabels: prometheus.Labels{"service": "provisioning"},
+		},
+		[]string{"type", "status", "error"},
+	)
+
```

Comment:
I think it would be a good idea to introduce new package internal/metrics where we would have all the metrics as exported variables.